tests/invalid_models_tests/test_relative_fields.py

```diff
@@ -559,9 +559,12 @@ def test_related_field_has_invalid_related_name(self):
             'contains_%s_whitespace' % whitespace,
             'ends_with_with_illegal_non_alphanumeric_%s' % illegal_non_alphanumeric,
             'ends_with_whitespace_%s' % whitespace,
-            # Python's keyword
-            'with',
+            'with', # Python's keyword
+            'related_name\n',
         ]
+        from django.utils import six
+        if six.PY3:
+            invalid_related_names.append(chr(65292))  # ord("，")==65292
```

Comment:
I tried add test for Python2. But there are some errors.

At line 570-573
`Child = type(str('Child_%s') % str(invalid_related_name)` got `TypeError: type() argument 1 must be string, not unicode` when there is `str(a unicode string)`.

I removed `str()`
`Child = type(('Child_%s') % (invalid_related_name), ....` and got `TypeError: type() argument 1 must be string, not unicode`